src/changelist/_config.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from github import Github, UnknownObjectException
+
+try:
+    import tomllib
+except ModuleNotFoundError:
+    import tomli as tomllib
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_CONFIG = r'''
```

Comment:
Why not store the default config as a toml file?

Reply:
Done in https://github.com/scientific-python/changelist/pull/30/commits/d56c6f271cf703f3fcfe50fdeeede39076edf8d1. I feel like having to add another config file (MANIFEST.in) makes it the less straightforward solution.

Reply:
Hmm, but that way it can be copied into the README as the default configuration. :D

Reply:
I think that's much better than having it inlined in the code. Why are you concerned about adding it to MANIFEST.in?